app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
```

Comment:
- [ ] Here we need to define an association between the Like model and the User model as we did in the `Post` model. It indicates that a like belongs to an author, and the author is an instance of the User model. Kindly fix this.

e.g
```
`belongs_to :author, class_name: 'User`
```